testament/tester.nim

```diff
@@ -12,10 +12,11 @@
 import
   parseutils, strutils, pegs, os, osproc, streams, parsecfg, json,
   marshal, backend, parseopt, specs, htmlgen, browsers, terminal,
-  algorithm, compiler/nodejs, times, sets, md5
+  algorithm, compiler/nodejs, times, sets, md5, sequtils
```

Comment:
But ``sequtils`` is pretty complex, if we break it, it's nice to see sequtils test case failures instead of broken testament builds.

Reply:
ok, will remove dependency on sequtils

Reply:
done